nixos/modules/services/backup/syncoid.nix

```diff
@@ -406,18 +350,28 @@ in
                 "~@privileged"
                 "~@resources"
                 "~@setuid"
-                "~@timer"
               ];
               SystemCallArchitectures = "native";
               # This is for BindPaths= and BindReadOnlyPaths=
               # to allow traversal of directories they create in RootDirectory=.
               UMask = "0066";
-            };
+            } //
+            (
+            if length sshKeyCred > 1
+            then { LoadCredentialEncrypted = [ c.sshKey ]; }
```

Comment:
I can see how this is useful, particularly since using `LoadCredential` for the SSH key somewhat precludes configuring `LoadCredentialEncrypted` manually, but it seems odd that no other service has implemented a similar pattern. I guess the feature is too new?

Reply:
You're right, I came up with it on my own. AFAICS it could become a pattern but is not yet.